Software-Engineering/L1-Junior-Software-Engineer.md

```diff
@@ -41,13 +41,16 @@
 - I show independent initiative and problem solving.
 - I demonstrate a collaborative attitude.
 - I take responsibility for my own work.
+- I avoid going dark for long periods of time.
+- I reach out to my peers for guidance and to ask questions when input is required to move forwards.
 
 <details>
 <summary>Examples</summary>
 
 - I found it difficult to set up my environment based on the instructions so I updated the instructions.
 - I didn't understand why our team wouldn't rewrite a difficult part of the codebase, so I asked my team lead to explain it to me.
 - A senior engineer on my team gave me some critical feedback on my work, so I took the time to understand how to improve.
+- I was working task and had a few options forwards, so I reached out to a team member to talk through the options and my thoughts, and to get their input to help choose a path to proceed.
```

Comment:
Nice. Slight suggestion for improved wording.

> I was working on a task and had a few options for moving forward, so I reached out to a team member to discuss the options and my thoughts. I used their input to help me choose a path to proceed.

Reply:
Thanks Matt, this was a really awkward sentence, I'll take the suggestion!